### ISSUE TITLE
IS THIS STILL MAINTAINED?

### ISSUE THREAD

**Issue by jmuheim** [winmarkltd/BootstrapFormHelpers#227]
I have asked the owner of this weeks ago already, and didn't receive a response. I think it's a big rip off to claim that paying users get support, but then nothing happens for weeks.

**Comment by j-steyn** [comment]
Wish I saw this before buying... :( Bought 'premium' support and when I tried to raise a support request the link does not even work. Fantastic customer service NOT.

**Comment by jmuheim** [comment]
Yeah, be careful and immediately stop your payment, as it will be re-paid every year automatically (I didn't even know about it anymore a year after and never was able to use it because of lack of support).

This whole thing should be shut down completely.

**Comment by niftylettuce** [comment]
Haha... why on earth would ANYONE in their right mind pay for something that is OPEN SOURCED ON GITHUB WITH AN APACHE LICENSE?  Support = read the documentation 😆
